README.md

```diff
@@ -1,27 +1,17 @@
-
 [![Build Status](https://travis-ci.org/drand/drand.svg?branch=master)](https://travis-ci.org/drand/drand)
 
 # Drand - A Distributed Randomness Beacon Daemon
-
-Drand (pronounced "dee-rand") is a distributed randomness beacon daemon written
```

Comment:
Sounds a bit harsh to remove it alltogether no ? I'd like to keep a very short intro of what drand actually is.

Reply:
Oops. Honest accident.